pkg/service/controller.go

```diff
@@ -124,9 +124,7 @@ func (c *ControllerService) CreateVolume(ctx context.Context, req *csi.CreateVol
 	dv.ObjectMeta.Annotations = map[string]string{
 		"cdi.kubevirt.io/storage.deleteAfterCompletion": "false",
 	}
-	dv.Spec.PVC = &corev1.PersistentVolumeClaimSpec{
-		AccessModes: []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce},
-		VolumeMode:  &volumeMode,
+	dv.Spec.Storage = &cdiv1.StorageSpec{
 		Resources: corev1.ResourceRequirements{
```

Comment:
i'm unsure about this. The `volumeMode` here comes from the csi request. I think we either have to be able to satisfy that request, or reject the request. By offloading the volumeMode to the cdi storage api, I'm not sure we can guarantee the volumeMode is honored.

Reply:
So this is the volume mode used in the DataVolume of the infra cluster, once hotplugged regardless of that volume mode, the disk.img or block device shows up as a block device on the tenant host. The node code will honor the request of the tenant user by putting a filesystem on that or not.